README.md

```diff
@@ -16,6 +16,13 @@ npm i react-kodyfire
 kody g react component
 ```
 
+![create your project with your file type](kody1.png)
```

Comment:
@Chandak-Keshav the explanation is a good idea but it's not part of the cli. `react-kodyfire` is a seperate package with its dedicated readme.

Reply:
Ok,will work on these right away.